internal/strace/strace.go

```diff
@@ -45,25 +46,27 @@ var (
 
 	// This regex parses just the file path. Bytes written is parsed further below as the nature of the write buffer makes it unideal to parse through regex.
 	// TODO: We can see how we can potentially reuse regex patterns.
+	// I0928 00:18:54.794008     365 strace.go:593] [   6:   6] uname E write(0x1 pipe:[5], 0x555695ceaab0 "Linux 4.4.0\n", 0xc)
```

Comment:
Remind me: are we extracting the full file path correctly for all of these writes? 

i.e. do these strace logs only give FDs, which we have to map to open/close calls to get the full file path?

Reply:
We are able to get the absolute file path for these writes since gvisor saves the strace fd argument with both the fd and the absolute path: https://github.com/google/gvisor/blob/master/pkg/sentry/strace/strace.go#L166-L173

Reply:
got it, thanks!